serverless/assets/verify_expiry.private.js

```diff
@@ -0,0 +1,15 @@
+exports.handler = function (context, event, callback) {
+  if (Date.now() > context.APP_EXPIRY) {
+    const response = new Twilio.Response();
+    response.appendHeader('Content-Type', 'application/json');
+    response.setStatusCode(401);
+    response.setBody({
+      error: {
+        message: 'passcode expired',
+        explanation:
+          'The passcode used to validate application users has expired. Re-deploy the application to refresh the passcode.',
+      },
+    });
+    return callback(null, response);
+  }
+};
```

Comment:
Curious why is this under assets and the rest are under functions?

Reply:
If you agree with my middleware suggestion, we can probably move this under `middleware` folder.

Reply:
Uploading JS files as assets allows us to `require()` them in our functions. This is what is recommended in [the docs](https://www.twilio.com/docs/labs/serverless-toolkit/general-usage#assets):

>Assets are a great way for you to also host resources that you want to access from your Functions that are not other Functions or should not be available to the public. For example configuration files or other JavaScript files you want to require(). You can store those as private Assets.

The same technique is used in the plugin-rtc repo, and both you and Aaron have suggested that the folder be named `/middleware`: https://github.com/twilio-labs/plugin-rtc/pull/55#discussion_r528928922

I think this makes sense. The file is technically an asset, but it functions as middleware, so I'll change the directory name.